app/policies/g5_updatable/location_policy.rb

```diff
@@ -0,0 +1,34 @@
+module G5Updatable
+  class LocationPolicy < G5Authenticatable::BasePolicy
+    class Scope < G5Authenticatable::BasePolicy::BaseScope
+
+      def resolve
```

Comment:
same here, 

I think you should be using the `clients_from_location_roles` method inside the actual definition of the `resolve` instead of using it externally